projects/plugins/crm/includes/ZeroBSCRM.List.Views.php

```diff
@@ -60,7 +60,7 @@ function zeroBSCRM_render_customerslist_page(){
                 'statusnotupdated' => __('Could not update statuses!',"zero-bs-crm"),
  
                 // bulk actions - contact deleting
-                'andthese' => __('Shall I also delete the associated Invoices, Quotes, Transactions and Events?',"zero-bs-crm"),
+				'andthese'             => __( 'Shall I also delete the associated Invoices, Quotes, Transactions and, Tasks?', 'zero-bs-crm' ),
```

Comment:
If we were intending to use the Oxford comma then it would need to go after Transactions, looks like it was just mis-placed here as it's added elsewhere.

```suggestion
				'andthese'             => __( 'Shall I also delete the associated Invoices, Quotes, Transactions, and Tasks?', 'zero-bs-crm' ),
```

Reply:
Yay! Thanks for ensuring the serial comma is present and proper. :^)